src/main/java/org/opensearch/knn/index/engine/MethodComponentContext.java

```diff
@@ -1,15 +1,9 @@
 /*
+ * Copyright OpenSearch Contributors
  * SPDX-License-Identifier: Apache-2.0
- *
- * The OpenSearch Contributors require contributions made to
- * this file be licensed under the Apache-2.0 license or a
- * compatible open source license.
- *
- * Modifications Copyright OpenSearch Contributors. See
- * GitHub history for details.
  */
 
-package org.opensearch.knn.index;
+package org.opensearch.knn.index.engine;
```

Comment:
Same with this, not engine specific

Reply:
Lets discuss on https://github.com/opensearch-project/k-NN/pull/1913#discussion_r1698810135